crate/content.md

```diff
@@ -1,90 +1,113 @@
-# What is Crate?
+# What Is Crate?
 
-Crate allows to query and compute data with SQL in real time by providing a distributed aggregation engine, native search and super simple scalability. It offers auto-sharding and replication, super-fast multi index queries, distributed aggregations and sort, superfast full-text search, and super simple cluster management.
+Crate is an open source fast, scalable, easy to use SQL database that plays nicely with containers like Docker. It feels like the SQL databases you know, but makes scaling and operating your database ridiculously easy - regardless of the volume, complexity, or type of data. It ingests millions of records per second for time series setups and delivers analytics results in sub-second real time.
+
+Crate comes with a distributed sort and aggregation engine, fast multi index queries, native full-text search and super simple scalability with sharding and partitioning builtin. Preconfigured replication takes care of data resiliency. The cluster management can be supervised with a built-in admin UI. Crate's masterless architecture and simplicity make the data part of Docker environments easy and elegant.
+
+Crate provides several installation packages, including a supported Docker image. It fits perfectly into an orchestrated microservices environment. It acts like an ephemeral, omnipresent, persistent layer for data. Application containers access their data regardless of which host the data nodes run.
 
 [Crate](https://crate.io/)
 
 %%LOGO%%
 
-## Crate on Docker
-
-Learn more about Crate and Docker and visit our [Docker page](https://crate.io/c/docker) at [crate.io](https://crate.io). You can also contact us on [Slack](https://crate.io/docs/support/slackin/).
+# Quick Start Example: Multihost Production Setup
 
-## How to use this image
+This is an example configuration to run in a multi-host production environment. The configuration includes the required minimum settings: - Volume mapping - Port mapping to localhost (run only one container per machine) - Unicast host discovery
```

Comment:
It looks like this was intended to be a bulleted list but became a run-on paragraph instead. :smile:
```markdown
This is an example configuration to run in a multi-host production environment. The configuration includes the required minimum settings:

- Volume mapping
- Port mapping to localhost (run only one container per machine)
- Unicast host discovery
```